src/index.ts

```diff
@@ -700,6 +733,9 @@ async function createGLTFNodes(gltfLoader, tile, unlitMaterial, options, rootTra
             if (options.computeNormals) {
               mesh.geometry.computeVertexNormals();
             }
+
+            mesh.receiveShadow = options.receiveShadow;
```

Comment:
yes, I just put that in there because I wanted to try out some shadows :-) Maybe the name of the property on the options object is not optimal, it should probably better reflect that fact that it is applied to the mesh?